src/codeflare_sdk/cluster/config.py

```diff
@@ -63,21 +70,54 @@ def __post_init__(self):
             )
         self._memory_to_string()
```

Comment:
```suggestion
        self._memory_to_resource()
        self._memory_to_string()
```
@ChristianZaccaria I think it could work better if we just place `memory_to_resource` before `memory_to_string` this _should_ solve the issue without repeating code WDYT?

Reply:
I already made the previous change christian suggested which solved the issue, should we use this approach instead?